Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System;
+
+namespace BootCamp.Chapter1
+{
+    public static class ArrayOperations
+    {
+        /// <summary>
+        /// Sort the array in ascending order.
+        /// If array empty or null- don't do anything.
+        /// </summary>
+        /// <param name="array">Input array in a random order.</param>
+        public static void Sort(int[] array)
+        {
+            if (array == null || array.Length == 0) return;
+            for (int i = 0; i < array.Length; i++)
+            {
+                for (int j = i + 1; j < array.Length; j++)
+                {
+                    if (array[j] < array[i])
+                    {
```

Comment:
This is a swap. You can extract this part to a function, as it is repeated also in reverse array.